lib/persistent-knex-model-mixin.js

```diff
@@ -27,7 +27,7 @@ module.exports = function PersistentKnexModelMixin (Model, knex, inOptions, user
     return this._applyFilter(this.constructor.convertToPersistent, this.getData())
   }
 
-  Model.prototype.save = function * (options) {
+  Model.prototype.save = function (options) {
```

Comment:
This was an unnecessary generator? This change doesn't affect any of the components, right?